SeaPublicWebsite/Views/EnergyEfficiency/BungalowType.cshtml

```diff
@@ -24,7 +25,7 @@
         
         <form action="@Url.Action("BungalowType_Post", "EnergyEfficiency", new {reference = Model.Reference, change = Model.Change})" method="post" novalidate>
             
-            @(Html.GovUkRadiosFor(m => m.BungalowType,
+            @(await Html.GovUkRadiosFor<BungalowTypeViewModel, BungalowType>(m => m.BungalowType,
```

Comment:
Now that this is building correctly can we delete the explicit typing (`<BungalowTypeViewModel, BungalowType>`) here and in all the other `GovUkRadiosFor` calls?

Reply:
Yup! Those are not needed anymore, I removed them.